docs/lisk-tools/node-providers.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: API Providers
-slug: /lisk-tools/api-providers
+title: Node Providers
```

Comment:
Unable to build and serve as when I run `npm run build && npm run serve`  I get the following:
<img width="1430" alt="image" src="https://github.com/user-attachments/assets/e68bead8-bab6-4e5a-9016-bf074f28aa0d" />

Reply:
errors should be fixed now!